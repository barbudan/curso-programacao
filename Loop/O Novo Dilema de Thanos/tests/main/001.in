3
100008
3817
17232