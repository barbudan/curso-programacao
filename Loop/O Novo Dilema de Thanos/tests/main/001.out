Thanos nao poupara o Planeta 1.
Thanos poupara o Planeta 2.
Thanos nao poupara o Planeta 3.
